### DISCUSSION TITLE
document LOG_LEVEL values

### DISCUSSION THREAD

**Discussion by digital88** [plausible/analytics#3570] [Self-Hosted Support] [answered]
Hello.

Could you please add LOG_LEVEL documentation here https://plausible.io/docs/self-hosting-configuration?
Setting this variable to unsupported value crashes app on startup.

**Comment by ruslandoga** [comment, marked as answer]
👋 @digital88 

I've opened a PR to add some info on `LOG_LEVEL` https://github.com/plausible/docs/pull/451
